PWGCF/FemtoUniverse/TableProducer/femtoUniverseProducerTask.cxx

```diff
@@ -1069,6 +1069,23 @@ struct femtoUniverseProducerTask {
     }
   }
 
+  template <bool isMC, typename V0Type, typename TrackType,
+            typename CollisionType>
+  void fillCollisionsAndTracksAndV0AndPhiCentRun3(CollisionType const& col, TrackType const& tracks, V0Type const& fullV0s)
+  {
+    if (ConfIsRun3)
+      fillCollisionsCentRun3<isMC>(col, tracks);
```

Comment:
Also do we need a separate function for that? The only difference I see between this one and fillCollisionsAndTracksAndV0AndPhi is the function for filling collisions? What if we would just add an if in the old function?